evap/staff/views.py

```diff
@@ -2217,9 +2217,10 @@ def template_edit(request, template_id):
         EmailTemplate.EDITOR_REVIEW_REMINDER,
         EmailTemplate.PUBLISHING_NOTICE_CONTRIBUTOR,
         EmailTemplate.PUBLISHING_NOTICE_PARTICIPANT,
-        EmailTemplate.TEXT_ANSWER_REVIEW_REMINDER,
     ]:
         available_variables += ["evaluations"]
+    elif template.name == EmailTemplate.TEXT_ANSWER_REVIEW_REMINDER:
+        available_variables += ["evaluation_url_tuples"]
```

Comment:
codecov says this is not covered. I don't think we need to have it covered, though. @niklasmohrin how much do we want those 96%?

Reply:
:shrug: its one line, I guess it's fine to leave it uncovered

anyways, why is codecov saying +0.42% on this PR? Is it against an old `main`? :thinking: